code/lib/util.js

```diff
@@ -26,11 +25,11 @@ function findItems(items, searchTerm) {
    var matches = []
    //a lot can be improved here to improve matching the items
    for (var i=0; i<items.length; i++) {
-      if (textLib.fuzzyMatch(items[i].shirt.title, searchTerm)) {
+      if (shirts[i.title].toLowerCase().includes(searchTerm.toString().toLowerCase())) {
```

Comment:
why was this changed from `items[i].shirt.title` to `shirts[i.title]`?

Reply:
oh I had this fixed but it must have not saved.